### PR TITLE
Docker support

### PR DESCRIPTION
**Pull request by ir4y**

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,9 +1,9 @@
 dj-database-url==0.3.0
 Django==1.8.13
-django-debug-toolbar==1.3.2
 django-jinja==1.4.1
 django-jinja-bootstrap-form==4.0.3
 funcy==1.7.2
+django-model-utils==1.3.1
```
Comment (Suor) [inline requirements.txt]: Why is that?

Comment (ir4y) [inline requirements.txt]: If we upgrade django-pandas to 0.4 there will no need of django-model-utils

It seems that we will need to upgrade to Django 1.9
https://github.com/chrisdev/django-pandas/search?q=model_utils&type=Issues&utf8=%E2%9C%93

Comment (Suor) [inline requirements.txt]: So, we don't require it directly, and by removing this line we don't need to remember to remove it after updating.

Comment (ir4y) [inline requirements.txt]: Yes, but the app fails to run if we don't install it.
So we have to add it in requirements.txt

Comment (Suor) [inline requirements.txt]: But doesn't django-pandas installs it [via setup](https://github.com/chrisdev/django-pandas/blob/0.3.0/setup.py#L25)?

Comment (Suor) [inline requirements.txt]: And you set too old version here, looks like it should be >=1.4

Comment (ir4y) [inline requirements.txt]: It install django-model-utils-2.6
At 2.4 PassThroughManager was removed from django-model-utils  https://github.com/carljm/django-model-utils/blob/68bc61e82517cb01fa36c0d007091a8670425913/CHANGES.rst#24-20151203

So it should be 
`django-model-utils>=1.4,<2.4`

**Pull request merged**